docs/source/dev-setup/devenv.rst

```diff
@@ -38,7 +40,7 @@ Once Homebrew is ready, installing the necessary prerequisites is very easy, for
     brew install git jq
     brew install --cask docker
 
-Go and SoftHSM are akso available from Homebrew but make sure you install the appropriate versions
+Go and SoftHSM are also available from Homebrew but make sure you install the appropriate versions
```

Comment:
punctuation. -- "Go and SoftHSM are also available from Homebrew, but make sure you install the appropriate versions."